### PR TITLE
TASK: Refactor absolute namespaces in Media / Domain

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: TYPO3.Media/Classes/TYPO3/Media/Domain/EventListener/ImageEventListener.php [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 use Doctrine\ORM\Event\LifecycleEventArgs;
 use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Cache\CacheManager;
+use TYPO3\Flow\Resource\Resource;
```
Comment (kitsunet) [inline TYPO3.Media/Classes/TYPO3/Media/Domain/EventListener/ImageEventListener.php]: Should be PersistentResource

File: TYPO3.Media/Classes/TYPO3/Media/Domain/EventListener/ImageEventListener.php [hunk 2]
```diff
@@ -36,7 +38,7 @@ public function postRemove(LifecycleEventArgs $eventArgs)
     {
         $entity = $eventArgs->getEntity();
         if ($entity instanceof ImageInterface) {
-            /** @var \TYPO3\Flow\Resource\Resource $resource */
+            /** @var Resource $resource */
```
Comment (kitsunet) [inline TYPO3.Media/Classes/TYPO3/Media/Domain/EventListener/ImageEventListener.php]: Should be PersistentResource

File: TYPO3.Media/Classes/TYPO3/Media/Domain/Service/ImageService.php [hunk 3]
```diff
@@ -78,7 +88,7 @@ public function injectSettings(array $settings)
      * @return array resource, width, height as keys
      * @throws ImageFileException
      * @throws InvalidConfigurationException
-     * @throws \TYPO3\Flow\Resource\Exception
+     * @throws Exception
      */
     public function processImage(FlowResource $originalResource, array $adjustments)
```
Comment (kitsunet) [inline TYPO3.Media/Classes/TYPO3/Media/Domain/Service/ImageService.php]: Could be chnaged to PersistentResource

**Comment by johannessteu** [comment]
fixed

**Comment by kitsunet** [comment]
Should be good now, merging when Travis is done.

**Pull request merged**